gui/_button.scss

```diff
@@ -1,5 +1,6 @@
 button,
-[role="button"] {
+[role="button"],
+input[type="file"]::file-selector-button {
```

Comment:
This causes a regression to the existing buttons, from the [preview deployment](https://7css-git-fork-lfuelling-file-input-khangs-projects-b59cdbd8.vercel.app/#button), you can see all buttons are no longer interactive when hovered, clicked etc.

I have played around with this a little bit and here are some discoveries:
- As a pseudo-element itself, `::file-selector-button` cannot nest other pseudo-elements like `::before` or `::after`, nor can it work like a normal button element.
- There are also no `:disabled` or `:focus` pseudo-classes, only `:hover`

Given, I'd suggest separating the file selector button to avoid giving the `postcss-nested` plugin grief and facing unexpected errors during build.

We may consider installing this [`postcss-extend-rule`](https://www.npmjs.com/package/postcss-extend-rule) plugin to reuse the applicable styles from the normal button.